src/main/webapp/app/entities/course/course.component.ts

```diff
@@ -5,6 +5,7 @@ import { JhiAlertService, JhiEventManager } from 'ng-jhipster';
 
 import { Course } from './course.model';
 import { CourseService } from './course.service';
+import * as moment from 'moment';
```

Comment:
See comment below, this is a pretty big import for the case that we only use one function.
I think it's worth it to do it in the template here.